tests/functional/conftest.py

```diff
@@ -0,0 +1,43 @@
+import subprocess
+
+import pytest
+
+
+@pytest.fixture(scope="session", autouse=True)
+def install_dcgm_snap():
+    """Install the snap and enable openstack-exporter service for testing."""
+    snap_build_name = "charmed-openstack-exporter_*.snap"
```

Comment:
I like this method, and nice that it replaces an otherwise required environment variable in the tests. If you have time, could you check our other snaps where we use the TEST_SNAP variable, and see where we can update the tests to this new pattern? Then we can remove the TEST_SNAP variable from the check.yaml template. :+1:

Reply:
I just noticed the name of the function :)

Reply:
Oh aha I didn't even see that :+1: